R/release.r

```diff
@@ -55,15 +55,9 @@ release <- function(pkg = ".", check = TRUE, args = NULL) {
   }
 
   if (uses_git(pkg$path)) {
-    if (git_uncommitted(pkg$path)) {
-      if (yesno("Uncommited changes in git. Proceed anyway?"))
-        return(invisible())
-    }
-
-    if (git_sync_status(pkg$path)) {
-      if (yesno("Git not synched with remote. Proceed anyway?"))
-        return(invisible())
-    }
+    git_checks(pkg)
+    if (yesno("Were Git checks successful?"))
```

Comment:
It's currently so you can ignore if needed — i.e. I don't always actually care that there are uncommitted files but I want to know about it.

Also, I don't actually care if there are local changes that haven't been pushed to the remote; but I _do_ care if there are remote changes I don't have locally. How hard would it be to make that work?